includes/pipeline_page/usage.php

```diff
@@ -112,9 +112,9 @@
                 $help_text = parse_md($vv['help_text']);
                 if(strlen($help_text)>150){ // collapse long help texts
                   $schema_content.='<div class="col-2">';
-                  $schema_content.='<button class="btn btn-outline-secondary" data-toggle="collapse" href="#'.preg_replace("/\/|\s/","-",$kk).'-help" aria-expanded="false"><i class="fa"></i> Details</button>';
+                  $schema_content.='<button class="btn btn-sm btn-outline-secondary" data-toggle="collapse" href="#'.preg_replace("/\/|\s/","-",$kk).'-help" aria-expanded="false"><i class="fas fa-question-circle"></i> Help</button>';
```

Comment:
maybe add a `pull-right` class, to the button, so it aligns with the "required" tag.

Reply:
Rewriting all of this code now 👍🏻